### ISSUE TITLE
Alternative source language not respected

### ISSUE THREAD

**Issue by saneinsaner** [typovision/l10nmgr#1]
When I choose an Alternative source language in L10N Manager it is not respected. 
Pressing "Refresh" or "Export" resets the dropdown to -default-.
When generating an excel file filename and content have default language.  (TYPO3 8.7.9)

**Comment by ltrebing** [comment]
tl;dr: This is not where the development of `l10nmgr` happens. Try the [Forge issue tracker](https://forge.typo3.org/projects/extension-l10nmgr/issues) instead.

This repository was both an attempt to make some sense of the mess that this extension’s Subversion repository was and a place for a few fixes and performance tweaks that we (Typovision GmbH, now a part of Portaltech Reply Germany) made in order to be able to use the extension for some of our enterprise customers. The last changes made in this repository will be four years old by the end of this month.

As far as I remember, the maintainers of the extension decided to squash our fixes (or some of them) into one Subversion commit and discarded the Subversion-to-Git conversion effort made here, opting to stick with Subversion instead. We stopped developing our own `l10nmgr` fork shortly afterwards because there was no need for it anymore, so this repository is kind of outdated by now.

I haven’t been following the recent development of the extension but based on a quick glance at [its Forge page](https://forge.typo3.org/projects/extension-l10nmgr) it seems to be in good hands with its current maintainers. Also I believe that the [Forge issue tracker](https://forge.typo3.org/projects/extension-l10nmgr/issues) is being actively monitored and used for development.
